### PR TITLE
[WIP] RFCs

### PR DESCRIPTION
**Pull request by turnrye**
This is an early draft of a proposed RFC repo and process. It's very very similar to what exists today on React. I'm still researching other RFC processes and trying to incorporate additional improvements. If there's any early feedback on this, feel free to share or even directly make a change if you're so compelled.

### CODE REVIEW

File: rfcs/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- Addition or removal of warnings
+- Additions that strictly improve objective, numerical quality criteria (speedup, better browser support)
+- Additions only likely to be _noticed by_ other implementors, invisible to users
+
+## What the process is
+
+In short, to get a major feature added to React Native, one usually first gets the RFC merged into the RFC repo as a markdown file. At that point the RFC is "active" and may be implemented with the goal of eventual inclusion into React Native.
+
+- Fork the RFC repo [react-native-community/react-native-releases](http://github.com/react-native-community/react-native-releases)
+- Copy `rfcs/0000-template.md` to `rfcs/0000-my-feature.md` (where 'my-feature' is the title in kebab case; don't assign an RFC number yet).
```
[the remark this answers is not in the document]
Comment (turnrye) [reply rfcs/README.md]: The RFC number isn't known until the PR is made; they wont know it yet.

File: rfcs/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The author of an RFC is not obligated to implement it. Of course, the RFC author (like any other developer) is welcome to post an implementation for review after the RFC has been accepted.
+
+If you are interested in working on the implementation for an 'active' RFC, but cannot determine if someone else is already working on it, feel free to ask (e.g. by leaving a comment on the associated issue).
+
+## Reviewing RFC's
+
+Each week the team will attempt to review some set of open RFC pull requests.
+
+We try to make sure that any RFC that we accept is accepted at the Friday team meeting, and reported in [core team notes]. Every accepted feature should have a core team champion, who will represent the feature and its progress.
```
[the remark this answers is not in the document]
Comment (turnrye) [reply rfcs/README.md]: Oops! Leak :)

**Comment by kelset** [comment]
Left a couple comments here, but overall 100% solid and I like it.

Side things:

1) should we then move the CHANGELOG into a `/changelog/` folder with a file per each major version? I feel it would help keeping clear the "separation" of the various elements of this repo

2) should we move all the For Discussions labelled issues on the main repo to this format? Would be a good way to get a pool of RFCs here right away (also to make it easier to understand which kind of subjects should be discussed via RFC)

3) Should the RFC folder also have a .md file with an overview of all the RFC open and maybe a quick "recap" of the proposal, or something like that?

**Comment by turnrye** [comment]
I'm comfortable with moving the changelog to a separate directory, however I do believe that it should be kept unified in one file. This is consistent with most other projects. I do think there's more progress to be made on streamlining the changelog though.

I like the idea of converting For Discussion to RFCs; I fear however that if we move them over for users, they may not feel ownership of them. I'd suggest instead incentivising users to move them.

The filtered PR link should show all of the open ones. I'm open to tracking statuses in another way, especially if it can include more context for the change (like the summary).

File: rfcs/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kelset) [reply rfcs/README.md]: Yes it was simply to "make more clear" that is was still an undefined number. No strong opinions anyway

**Comment by turnrye** [comment]
Got input out of Github that we should consider this less of a strict RFC approach and instead as broader planning and proposals. Going to do some revs tonight with that in mind.

**Comment by turnrye** [comment]
I didn't get through changes to the proposal template, but I did update a draft of the README this evening.

**Comment by kelset** [comment]
let's close this in favour of https://github.com/react-native-community/discussions-and-proposals ?

**Pull request closed without merging**